geospaas_processing/downloaders.py

```diff
@@ -99,8 +97,25 @@ def get_remote_file_size(response, auth):
         return file_size
 
     @classmethod
-    def download_url(cls, url, download_dir, file_prefix='', **kwargs):
-        """Download file from HTTP URL"""
+    def get_file_name(cls, response, url, file_prefix=None):
+        """ Get the name of downloaded file either from header of <response>, or from <url>.
+        Prepend the file name with <file_prefix> if given """
+        file_name = cls.extract_file_name_from_response(response)
+        if not file_name:
+            file_name = url.split('/')[-1]
+        if file_prefix:
```

Comment:
there is no need for this test, the next line takes care of it

Reply:
Done in the next commit